easybuild/easyconfigs/c/CondaCreate/requirements.txt

```diff
@@ -0,0 +1 @@
+perl-app-cpanminus
```

Comment:
I think we should avoid or at the very least discourage people to let conda install the latest version, part of the idea behind EasyBuild is that you can easily reproduce previous installations...